src/sinks/gcs_common/service.rs

```diff
@@ -97,10 +97,12 @@ impl Service<GcsRequest> for GcsService {
     type Future = BoxFuture<'static, Result<Self::Response, Self::Error>>;
 
     fn poll_ready(&mut self, _: &mut std::task::Context<'_>) -> Poll<Result<(), Self::Error>> {
+        // Emission of Error internal event is handled upstream by the caller.
         Poll::Ready(Ok(()))
     }
 
     fn call(&mut self, request: GcsRequest) -> Self::Future {
+        // Emission of Error internal event is handled upstream by the caller.
```

Comment:
```suggestion
        // Emission of Error internal event is handled upstream by the caller.
        
```

Being extra nitty, we've been linebreaking here 😛